docs/release-notes/index.md

```diff
@@ -21,8 +21,153 @@ To check for security updates, go to [Security announcements for the Elastic sta
+
+**Alerting**:
+* Enables incremental human-readable case IDs [#238555]({{kib-pull}}238555).
+* Create case analytics indexes per solution per space [#234125]({{kib-pull}}234125).
+* Shows thumbnail image of attached case file [#226004]({{kib-pull}}226004).
+* Adds a new Jira Service Management connector [#235408]({{kib-pull}}235408).
+* Adds `xpack.actions.email.recipient_allowlist` alert action setting, which lets you specify a list of allowed email recipient patterns (`to`, `cc`, or `bcc`) that can be used with email connectors [#220058]({{kib-pull}}220058).
+* Adds support for OAuth 2.0 authentication to the Webhook connector [#218442]({{kib-pull}}218442).
+* Allows you to specify additional fields for the IBM Resilient connector [#236144]({{kib-pull}}236144).
+* Maintenance windows are now generally available [#233870]({{kib-pull}}233870).
+* Adds support for encrypted headers in the Webhook connector [#233695]({{kib-pull}}233695).
+* Automatically copies source data into the alerts-as-data documents for other {{es}} query rule types [#230010]({{kib-pull}}230010).
+* You can now add filters to maintenance windows based on alert fields from all solutions, without needing to select a category first [#227888]({{kib-pull}}227888).
+
+**Dashboards and Visualizations**:
+* Adds the ability to configure index settings when importing geospatial files in Maps [#232308]({{kib-pull}}232308).
+* Adds a new inline Markdown editor to edit your text while switching between **Editing** and **Preview** modes [#229191]({{kib-pull}}229191).
+* Updates the toolbar popover for Metric charts, which is now called **Appearance** [#233992]({{kib-pull}}233992).
+* Improves ES|QL suggestions logic when an {{esql}} visualization query changes [#231767]({{kib-pull}}231767).
+* Adds a new **Add** menu to the toolbar that lets you populate your dashboard with visualizations, panels, sections, and controls [#230324]({{kib-pull}}230324).
+* Improves perceived performance of dashboard flyouts [#226052]({{kib-pull}}226052).
+* Adds the **Select all** functionality for Options list controls [#221010]({{kib-pull}}221010).
+
+**Data ingestion and Fleet**:
+* Adds a toggle on the {{es}} output configuration page that grants agents permission to write to the `logs` and `logs.*` data streams [#233374]({{kib-pull}}233374).
+* Completes OTel configuration pipelines with an exporter [#233090]({{kib-pull}}233090).
+* Enables feature to auto-install content packages when data is ingested for datasets that match the datasets defined in the packages' `discovery` fields [#232668]({{kib-pull}}232668).
+* Adds support for indexing package knowledge base docs into the `.integration_knowledge` system index, allowing AI assistants used by integration developers to have greater context relevant to specific packages [#230107]({{kib-pull}}230107).
```

Comment:
we decided to pull integrations knowledge support from 9.2 today (https://github.com/elastic/kibana/pull/239802), please remove this entry. thank you!